Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -555,11 +549,8 @@ private void OnDisable()
                 RegisterGlobalInputHandler(false);
             }
 
-            // If the gameObject or script is disabled we need to also internally disable Interactable
-            SetDisabled(true);
-
-            // Set the states before object or script is disabled
-            InternalUpdate();
+            // Set all base states to false when the gameObject or script is disabled
+            ResetBaseStates();
```

Comment:
Removed SetDisabled and InternalUpdate because they were causing a test failure for TestDestroy in InteractableTests.